gui/default/syncthing/core/syncthingController.js

```diff
@@ -3081,7 +3081,13 @@ angular.module('syncthing.core')
                 return false;
             }
             var counts = $scope.model[folderCfg.id];
-            return counts && counts.receiveOnlyTotalItems > 0;
+            if (!counts) return false;
+            if (folderCfg.type == 'receiveencrypted') {
+                // Deleting unexpected items won't trigger an index update.
+                // Subtract the deleted items from the total changed items.
+                return counts.receiveOnlyTotalItems - counts.receiveOnlyChangedDeletes > 0;
```

Comment:
It should be updated though? Reverting or manually deleting should replace the locally changed items with non-locally changed items in the db (from revert and/or scan), which in turn updates the counts. And that should reach the UI through the folder summary events. If that's not happening, we should fix that instead.